presto-iceberg/src/main/java/io/prestosql/plugin/iceberg/IcebergMetadata.java

```diff
@@ -598,23 +608,47 @@ public void rollback()
     @Override
     public Optional<ConstraintApplicationResult<ConnectorTableHandle>> applyFilter(ConnectorSession session, ConnectorTableHandle handle, Constraint constraint)
     {
-        IcebergTableHandle table = (IcebergTableHandle) handle;
+        IcebergTableHandle tableHandle = (IcebergTableHandle) handle;
```

Comment:
Let's keep the name the same so that the diff is smaller

Reply:
Done.